src/debugger/adapter/DebugAdapter.js

```diff
@@ -81,13 +81,15 @@ class PrepackDebugSession extends LoggingDebugSession {
 
   launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments): void {
     // set up the communication channel
-    this._adapterChannel = new AdapterChannel(args.inFilePath, args.outFilePath);
+    this._adapterChannel = new AdapterChannel(args.debugInFilePath, args.debugOutFilePath);
     this._registerMessageCallbacks();
     let launchArgs: PrepackLaunchArguments = {
       kind: "launch",
-      prepackCommand: args.prepackCommand,
-      inFilePath: args.inFilePath,
-      outFilePath: args.outFilePath,
+      prepackRuntime: args.prepackRuntime,
```

Comment:
If you can match the property names, much cleaner to use spread operator "...args".